sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/EvaluationOutcome.java

```diff
@@ -24,6 +22,7 @@
 
     private final Type type;
     private final String source;
+    private final MesosResource mesosResource;
```

Comment:
I like this.  I'll put it in a separate PR.  Really want to get this in.  It's blocking the PortsSpec bug fix, and I want to unblock @paegun, ASAP as well.